src/Spice86.Core/Emulator/InterruptHandlers/Dos/Ems/EmmMapping.cs

```diff
@@ -0,0 +1,32 @@
+namespace Spice86.Core.Emulator.InterruptHandlers.Dos.Ems; 
+
+public record EmmMapping {
+    public const int Size = 4;
+    private byte[] _data { get; init; } = new byte[4];
+    
+    public ushort Handle {
+        get => (ushort) ((_data[0] & 0xFF) | ((_data[1] & 0xFF) << 8));
```

Comment:
Since data is aleady an array of byte, why doing an and FF?

Reply:
This code has been removed.